src/ert/config/summary_config.py

```diff
@@ -30,19 +30,6 @@ def __post_init__(self) -> None:
     def read_from_file(self, run_path: str, iens: int) -> xr.Dataset:
         filename = self.input_file.replace("<IENS>", str(iens))
         _, keys, time_map, data = read_summary(f"{run_path}/{filename}", self.keys)
-
-        if self.refcase:
```

Comment:
Can remove this property?

Reply:
Then we have to change storage?

Reply:
I will create an issue for it

Reply:
Issue here: https://github.com/equinor/ert/issues/7015